### DISCUSSION TITLE
Security Onion only for HIDS

### DISCUSSION THREAD

**Discussion by networklord** [Security-Onion-Solutions/securityonion#13435] [General]
Hi all,

As  there is no possibility to mirror Network traffic between VMs inside ESXI I need advice  for distributed Install as I want to cover HIDS at least:

1.) Install Manager
2.) Install Search Node
3.) Possibly Install Fleet node ?

Would you also go with this setup?

Thanks.

**Comment by TotieBash** [comment]
What do you mean "As there is no possibility to mirror Network traffic between VMs inside ESXI"????  
For single ESXi you can use standard or distributed vswitch "promiscuous mode".  This behaves like a dumb hub.  
For multiple ESXi in a cluster,  you need distributed vswitch "mirroring" feature.

**Reply by networklord** [reply]
I have multiple VMs and one VM as Proxy which allows machines to access Internet. Are you saying that if I enable promiscuous mode on port group and add Security Onion forwarder to same group it will be able to monitor Network Traffic sent to Proxy and to VMs back?

**Reply by TotieBash** [reply]
Yes.  If you enable "promiscuous mode" on that specific port group and SO-forwarder "monitor" interface is on the same port group your SO will get a copy of every packet in/out of that particular port group.

**Comment by networklord** [comment]
Thanks very much for info!!!!!!

**Reply by TotieBash** [reply]
no problem...  Just to be super super clear, your proxy-vm and SO-vm needs to be on the same ESXi for "promiscuous mode" to work.

**Reply by networklord** [reply]
Understood. Only thing what confuses me is that promiscuous mode can be enabled on vswitch  lvl,  port lvl and NIC lvl.

My logic for this is:
If enabled on vswitch level it will accept traffic destinated to other vswitches and as default security policy say on port group inherit from vswitch. VMs will see each other traffic maybe even traffic from other port groups ( just putting theory here)
On nic level or uplink it will accept traffic destinated to other destinations in physical networks
-----------------------------------------------------------------------------------------------------------
Enabling it just on portgroup will do job for me
